rules/override-branch-protection/rule.md

```diff
@@ -0,0 +1,38 @@
+---
+type: rule
+title: Do you know when to override branch protection rules?
+uri: override-branch-protection
+authors:
+  - title: Matt Goldman
+    url: https://www.ssw.com.au/people/matt-goldman
+created: 2022-11-18T19:38:28.898Z
+guid: 65969915-ea02-43b7-a912-ca6e09c50e98
+---
+You should be using [branch protection](https://ssw.com.au/rules/use-branch-protection) rules to protect your main branch in your source repository. These can include minimum number of reviewers, specific required reviews, passing tests, or code quality checks. These help you maintain quality standards in your product, but you can override these checks with a privileged account and, sometimes, you should.
+
+<!--endintro-->
+
+Most source code repositories (including Azure DevOps and GitHub) allow you to set branch protection rules, meaning code must meet a set of conditions before it can be merged into the main branch. But administrators, or individuals with the appropriate delegated authority, can bypass these checks and merge code without these conditions being met.
+
+![Figure: A pull request in GitHub that has not met the requirements of the branch protection rule. However, as the person viewing this is a repository administrator, an option is available to bypass the branch rules and merge the code.](bypass-rule.png)
+
+### When SHOULDN'T you override branch protection rules?
+
+Just because you hold the keys, doesn't mean that you should infer a license to merge code at your own behest when it overrides protections that have been put in place for a reason.    
+    
+For example, if a PR is blocked because a reviewer has requested changes, you shouldn't use your privileges to override the branch protection rules to avoid making the requested changes. Just like with any other privileged account, with great power comes great responsibility, and it's _your_ responsibility to use those privileges for the purpose they were granted, and not for your own convenience.
```

Comment:
```suggestion
For example, if a PR is blocked because a reviewer has requested changes, you shouldn't use your privileges to override the branch protection rules to avoid making the requested changes. Just like with any other privileged account, with great power there must also come great responsibility, and it's _your_ responsibility to use those privileges for the purpose they were granted, and not for your own convenience.
```